docs/source/build_network.rst

```diff
@@ -156,7 +156,7 @@ Next, you can bring the network up with one of the following commands:
 
 The above command will compile Golang chaincode images and spin up the corresponding
 containers. Go is the default chaincode language, however there is also support
-for `Node.js <https://fabric-shim.github.io/>`_ and `Java <https://hyperledger.github.io/fabric-chaincode-java/>`_
+for `Node.js <https://hyperledger.github.io/fabric-chaincode-node/>`_ and `Java <https://hyperledger.github.io/fabric-chaincode-java/>`_
```

Comment:
Can this be changed the release specific link: https://hyperledger.github.io/fabric-chaincode-node/release-2.0/api/

Reply:
I would rather keep the link to the top level doc for the Node chaincode here rather than straight to a release specific api doc, as for the equivalent Java chaincode link. (Where there were links to specific APIs I did use the release specific urls.)